shardingsphere-infra/shardingsphere-infra-common/src/main/java/org/apache/shardingsphere/infra/metadata/schema/builder/TableMetaDataLoaderEngine.java

```diff
@@ -58,8 +74,14 @@
     public static Collection<TableMetaData> load(final Map<String, Collection<String>> dataSourceTable, final DatabaseType databaseType,
                                                  final Map<String, DataSource> dataSourceMap) throws SQLException {
         Optional<DialectTableMetaDataLoader> dialectTableMetaDataLoader = findDialectTableMetaDataLoader(databaseType);
-        return dialectTableMetaDataLoader.isPresent() ? loadByDialect(dialectTableMetaDataLoader.get(), dataSourceTable, dataSourceMap)
-                : loadByDefault(dataSourceTable, databaseType, dataSourceMap);
+        if (dialectTableMetaDataLoader.isPresent()) {
+            try {
+                return loadByDialect(dialectTableMetaDataLoader.get(), dataSourceTable, dataSourceMap);
+            } catch (SQLException e) {
+                return loadByDefault(dataSourceTable, databaseType, dataSourceMap);
```

Comment:
@tuichenchuxin Please print out the exception information and modify the exception variables.